maintainers/maintainer-list.nix

```diff
@@ -20222,6 +20222,13 @@
     githubId = 71843723;
     keys = [ { fingerprint = "EEFB CC3A C529 CFD1 943D  A75C BDD5 7BE9 9D55 5965"; } ];
   };
+  thepuzlemaker = {
```

Comment:
```suggestion
  thepuzzlemaker = {
```

Reply:
Good catch! I completely missed that :sweat_smile: